types/spectest/generate/tests.json

```diff
@@ -1 +1 @@
EFvvpZuleUaNbp409xOT3VR4nka8A1QNiSTaX1V6mcYui8IntozDiYQD7Wtu5Oa+vdJgnlUFEInEuhWDOw3Lq8aVrULpOAOQrFFU36U6I7d19IA0BU+223MEusphmawrvnqSV/M/aBrev/nInvOtB2ayd0VDfhkLNsR31q5K2lim2pczce6krbPEhciFwV9KRsv3G8QToZhe+G3//6zavWaAY9QwFXiTvm22v+DkIO8nf6ukQH2r/SfgI9gODSALRYCg81xsnW/l+tJ9WEiFcyP6Th1+ggq21H/Dcp1/1e6eFLXlChNtbjUmAAqgh5fwsV6J8wf5ZHxKGCJWwBXvHXsv/mCS7RuKvBqeFtq36njL0nWI1d2w7ys4YszDOwRJqwrtfNnkzkDeBfqtTb+0p/lxi73kMt0iT8VJCgLARHiT8AXITDQCog2HmNRHrVqluVS0CxJRFRMGTGJoVKETKScra44t0JEJqdNYXY3FqBoulyp873C4Om2RPHy4CWWvT9Ea7nxPfsYrSyaK6l3cbuZT4Aa/+Lo3ajWOONmxbgmPMiRqKNeUsefCFa/6vBxm9G/tUrMRneDtsCOVUkdObIPIYNCmROBw401ekZZuvjUTd9QRdfxFrzFX3jCig3BANBt/0TmObjnrflngWwD3lTszfn2QCooBoidLNmA805Rl3cfE/H6a4x8cyAxZku2df8S1kKuYkWekrHMKMYjSeFjaFC4o/9BHh8UsJf3+jsj6qFz0XoT8HA+7Nc1iqYjBXMl7tOBoroTxQoTo9itqtApWWDZJXCdbnyxAeiU2a6Ns9y4KkVXC24Cq2iqTqlPh3mpxFpVmXHqPp7HrIoUR1PsOOeEAbFLSJp5Jm3QUnpFBa3KWE/nQG2ffwXvRtJiOE+/HAYH90WkBoHUhVqjTDc3Wyz3yZ1Gtq5OOqnuIJeC3OnRZ7v/eWhuWUJqBRO5UYGO4Cqepe2LzQ+ZH0bu5ExugvxtB4I6n0Ty/Xv6DiUNZpnPetJXfrnq8N2bFZXPAYODw9RZVuUI+pgru3dEUioDusXLoi9YpBgB9XlDQSb1uGbcxsoEVOm+HH8vZkklTzzAkUIGj0EtXUVLTk1bVORZcZVQ8t8ZAaGEZ9nVKX7Uqe7bn0AvK/1L0uUKtpfqW+9efoCCZQuCNjWk8PVcGHEtD002WCTHmCfUVEJa7AtOplYc4/HFQRq03/JuJBJ5HNqii/bI+lOvQSgoxZnXh2UI948sgu5n6DV5R8aEivFD/F0gQJBJklzRsZQkRGZxHOfDSnKhZTklZLluKAQGqV2pJ9FOu7a5me9EbVrUmIHCGdrIrQLZlOBZVp6RuE8hGkw5o/vVlKjINapZduDIiZ2BpauqFDAWYrnhT7luqJhirYmM19d7zSVH8PQEcfhtSk8m5CdPLpX9mytgTehnvpVjCx6mzkW6eazIG5hmBeRqzaIIvTMC/8xug/kb/4Nis/lkHKAifYsxNB4yAAMrKs8efgQ7LDjqF1Cmwx8XSwqLABN+w+6iTtn9+Xm/BO2SOiz1zAWprPaX0nzNsqkDpIRnKc2bLurUFNqYPU9b/Q/08ufrdZiOWMOmNcJxzgw7MRQ/HWiYfeqshqHged6wez0tSX3l6+jZRIbpp7MAxpFiGmizr0/ueBx8BakxEj+RAFTQlsLhVJUNMqJsOO23DdpQokK+TRXOYMJldnsUEBHNhKpYXDr3mPwetepj6T4KQmw+FGj0AvfmTyAoGkvXPMEjQXTxdiqaQZifVwmXA2yIWx/b+cgVOh0Zr7ZSahI8pqL+bpjGAJ+EOfa27qiBRTzFjvNEM4/9BHg86cKMLjc4EqZRV2Q/Z57ZmrNfQCTm7jGHflNrA8OGFvzMmTNlFD3a/845yAWzkWdAcOmTxkZBVgQ6hCZoYMdp3iGLtfVpj0x6m3QUJTXLzAjVs/dHy/an37tsewzMWK9IhrxEFVhJbpyE2AZgEXd38BzbhMCi07Dy2Kbqz/Wg5VvRvmOHw3k66PUjHFlpeukUiUpJs64To6Eky/bOozt+tXXPbNE+YHPd9NAzo/hzZsJwia/ANwfvmkTIKDiHM9cPCbwI3ldNB88ZPwomoIzoJTv9sm1TBmMgEsba4ZR4PesxtyrDW71X8H8WZ3RthfXVshMriF+MsCBpSXgPlAYwc5bcfs55OCJfxaVcZe02CMJ/ZoupsIh1l54kllXRWkw9d7FDOrJvVtgOLKkXjYhQHuaCoU0HsXY16ZaNotmKqetKWuY55C7tDHc1MT0zCP0HlYnVyy3Tgakpj2cyWXam3NPjj4NuS8MQaIX2NIxXyQvq3Xy8oXVwuQEwqq3H6w7Sv7l8XIp5HQtrJzbUOw9ESQlEnm6jK6cG3f4oxAeoXoLj8iBkaW0v17N7M+AeABatkQR/lfDPXF1qvI/XaYxBVcKQoaHbhC535lbGmti8Bs6p4A2tvsiGymSvX+MEXryMVJvyPnHGNPAtqLJQYYwWmlTkr0XXmfLeZ0e94BOVoT8aYF6b5ZztBKUed+0HMPNCBXEWTqUkemcLli6/a0U2YHSMpQIAAAAAAAAAAwAAAAAAAAAAAQIDBAUGBwgJCgsMDQ4PEBESEwAAZKeztuANAgAAAAAAAAC4m+vGmXaXJqMYyOmXG9MXEpfGGupKZXinpPlLVH3LpbrBaokQi2tqH+NpXRqHSgsAAQIDBAUGBwgJCgsMDQ4PEBESEwABAgMEBQYHCAkKCwwNDg8QERITFBUWFxgZGhscHR4fICEiIyQlJicoKSorLC0uLzAxMjM0NTY3ODk6Ozw9Pj9AQUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVpbXF1eXw==","ExpectedRoot":[214,6,133,147,123,255,148,105,121,81,22,60,70,207,145,67,235,242,216,185,1,10,169,67,203,52,170,87,177,181,68,232],"ExpectedError":""}}
```

Comment:
I think nothing in this file changed besides the key? so you can revert this change

Reply:
Reverted 👍